src/lib/Scenes/Fair/Components/HoursCollapsible.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Flex, Sans, space } from "@artsy/palette"
+import { WhiteButton } from "lib/Components/Buttons"
+import React from "react"
+import styled from "styled-components/native"
+
+const Button = styled(WhiteButton)`
+  padding-left: ${space(2)};
+  padding-right: ${space(2)};
+`
+
+interface Props {
+  hours: string
+}
+
+interface State {
+  isExpanded: boolean
+}
+
+export class HoursCollapsible extends React.Component<Props, State> {
+  state = { isExpanded: false }
+
+  handleToggleIsExpanded = () => {
+    this.setState({ isExpanded: !this.state.isExpanded })
+  }
+
+  render() {
+    const { isExpanded } = this.state
+    const { hours } = this.props
+    return (
+      <Box>
+        <Flex justifyContent="space-between" alignItems="center" flexDirection="row">
+          <Sans size="4">Hours</Sans>
```

Comment:
Noting that this field IRL often [contains Markdown](https://user-images.githubusercontent.com/140521/48497302-dd499b00-e801-11e8-9339-a4f7a50d18e8.png) — do we have a pattern for rendering Markdown in RN views?

Reply:
@l2succes good q — I'm not sure if we have structured data for fair hours — will investigate

Reply:
We do have a markdown renderer in here somewhere, yep

Reply:
Nice catch, I pulled the markdown renderer up out of [`Bidding/Components`](https://github.com/artsy/emission/pull/1223/commits/3804c06acde7435f165e1a2f45ddc9a9c19b03c0) and am using it to render this text now.